CONTRIBUTING.md

```diff
@@ -1,51 +1,150 @@
 # Contributing
 
-1. If code changes are being made, please include documentation **following the guidelines in the Wiki** (typings are also welcome, but we can probably do them for you, given the documentation).
-2. (For the maintainers of this repo) Label PRs appropriately.
-3. Link relevant issues.
-4. Ensure compatibility with the latest changes in the repo.
-5. After you've made your changes, please test them out thoroughly. If the repository has any example code or code exclusively used for testing, please test your changes against those. If you have added new features, consider writing and contributing to a test for that feature to ensure it works properly.
+Thank you for contributing to Manim!  However you have decided to contribute or
+interact with the community, please always be civil and respect other members
+of the community.  Manim is a free open source software for mathematical
+animations, and as such we welcome **everyone** who is interested in
+mathematics, pedagogy, computer animations, open source, software development,
+and beyond.  Manim accepts contributions of many kinds, detailed below.
+
+
+## Reporting bugs
+
+One of the best ways of contributing to Manim is by reporting bugs.  If you
+have encountered something that you believe is a bug, please follow these
+steps:
+
+1. Make sure to search for other users who may have had similar issues in the
+   past.  Search the repository's issues page (don't forget to search closed
+   issues), bring it up on Discord, use sites like StackOverflow, and exercise
+   your best Google practices.  If you can't find anything helpful, then go to
+   the next step.
+
+1. Make sure you can reproduce the issue, i.e. that you have some code that
```

Comment:
Issue : Indexation mistake

Reply:
You can actually use any numbers for lists in markdown, and they will be displayed correctly:

1. this list
1. has all indices
1. equal to one

Reply:
i mean, sure, but as a md file this looks weird and inconsistent, like, either 1. 1. 1. 1. or 1. 2. 3. 4. ,  not 1. 1. 2. 3. 4. as that has no apparent pattern

Reply:
Ok, done!

Reply:
> You can actually use any numbers for lists in markdown, and they will be displayed correctly:

TIL that :D